libraries/wallet/wallet.cpp

```diff
@@ -1066,6 +1066,14 @@ class wallet_api_impl
 
       return true;
    }
+
+   void quit()
+   {
+        ilog( "Quitting Cli Wallet ..." );
+
+        throw fc::canceled_exception();
```

Comment:
Can't we use std::terminate() or something like that instead?

Reply:
@satyakoneru - please revert on this.

Reply:
Just a couple of reasons,
1. Took the fix as-is from bit-shares, if you see the function run of the async thread, it is broken by exceptions. I just followed the same approach for the new command as well.
2. As it's in bit shares from long back, it would have been tested fairly by now.